src/service/handlers/messages.js

```diff
@@ -38,6 +39,50 @@ class Message {
                 res.send(response);
               })
               .catch((err) => console.log(err));
+          break;
+        case 'meaning':
+          SCRAPING_URL =
+            'https://www.oxfordlearnersdictionaries.com/definition/english/' +
+            queryParam;
+          axios
+              .get(SCRAPING_URL)
+              .then((resd) => {
+                resd = resd.data;
+                if (resd) {
+                  const $ = cheerio.load(resd);
+                  $('ol > li').each(function() {
+                    const currentNode = $(this);
+                    const defNode = currentNode.find('span.def');
+                    let def;
+                    let example;
+
+                    if (defNode) {
+                      def = defNode.text();
+                    }
+
+                    const exampleNode = currentNode
+                        .find('ul.examples > li')
+                        .first();
+                    if (exampleNode) {
+                      example = exampleNode.text();
+                    }
+
+                    if (def) {
+                      const details = new Object();
+                      details.Meaning = def;
+                      details.Example = example;
```

Comment:
Rest looks good to me. Only update the keys `Meaning` to small letters as `meaning`, as per the convention to name the keys in the JSON object, and same for `Example` too.

Reply:
Should I do the same for Google search also?

Reply:
Sure.